pkg/agent/const.go

```diff
@@ -60,8 +60,6 @@ const (
 	kubernetesKubeletMonitorSystemdService    = "linux/cloud-init/artifacts/kubelet-monitor.service"
 	kubernetesDockerMonitorSystemdTimer       = "linux/cloud-init/artifacts/docker-monitor.timer"
```

Comment:
we can probably also get rid of the docker monitor stuff as well, maybe a good follow-up item